.tekton/retasc-pull-request.yaml

```diff
@@ -232,7 +232,7 @@ spec:
         - name: name
           value: buildah
         - name: bundle
-          value: quay.io/redhat-appstudio-tekton-catalog/task-buildah:0.1@sha256:6b483e420a9abc7462dd0a4886a520bf44fb8d1cb558badf30a67af0d2fd21fc
+          value: quay.io/redhat-appstudio-tekton-catalog/task-buildah:0.1@sha256:102500165339bc08791775cf2c4dcae3dd4bde557a9009d44dc590ef66dde384
```

Comment:
What's up with these changes? Feel free to merge any open PRs from Konflux separately (if they pass CI).

Reply:
I feel this is not from me, maybe got from some rebase action

Reply:
https://github.com/release-engineering/retasc/blob/main/.tekton/retasc-pull-request.yaml#L235, see here, it is same value

Reply:
I can still see the commit, but it seems to contain unrelated changes. Can you squash the commits?

Reply:
not sure how to do the git squash, in gitlab I normally do this action before merge in UI.

Reply:
squashed